lib/Installation/Partitioner/Libstorage/ExpertPartitionerController.pm

```diff
@@ -132,11 +132,16 @@ sub add_raid {
     $self->_finish_partition_creation();
 }
 
-sub accept_changes {
+sub accept_changes_and_press_next {
```

Comment:
Why not reuse `accept_changes` method. It's just one line duplicated, but still duplicated.

Reply:
Because of the design. Some of the controllers' methods because of the previous flow implemented. So in some cases they expect to press accept button and then next to continue to the rest of the installation. For the resize method as it is the flow differs as it ends with pressing the next button and such you need only to accept the changes to continue. 
I think that we might face this in another case in the future. so i try to do it as flexible as possible and avoiding hacks. I could put the logic in the existing `accept_changes` but i am afraid i would have to use some extra needle checking or variable. it is better to decouple IMO.

Reply:
The reason why it ends with just pressing 'Accept' button is because you've kept `partitioning_finish`, which is redundant. We do not follow this convention in other modules. 

If you are keeping a module which just presses "Next" button and does not test anything, maybe we need to do this for all other tests? Create 'press_next' module and call it after each testing module. Only in that case I see a potential possibility to use such approach.

Reply:
@OleksandrOrlov made clear to me that you are asking to reuse the accept_changes inside the method was what you were asking. thanks man.

Reply:
Sure, we can keep it the way it is.

Reply:
No i did the change already. actually i had it in my mind but just forget it.